commit 6565cf9b0b322b3a42886a748815d59d6af619cc
Author: Gijom <guillaume.chanel@unige.ch>
Date:   Tue Jun 6 10:24:35 2023 +0200

    Include cstdint (#37)
    
    This makes uint64_t and similar uint*_t types available.

diff --git a/xdf.h b/xdf.h
index 86892f3..47e73a1 100644
--- a/xdf.h
+++ b/xdf.h
@@ -26,6 +26,7 @@
 #include <vector>
 #include <map>
 #include <set>
+#include <cstdint>
 
 /*! \class Xdf
  *
